languages/elixir/exercises/concept/remote-control-car/.docs/instructions.md

```diff
@@ -63,38 +61,34 @@ RemoteControlCar.display_battery(car)
 
 Implement the `RemoteControlCar.drive/1` function that:
 
-- updates the number of meters driven
-- updates the battery percentage
+- updates the number of meters driven by 20
+- drains 1% of the battery
```

Comment:
I find it a bit annoying if for every step I need to scroll back to the beginning of the introduction to actually know what to do. I think in this case it's worth it to repeat this information here, in the specific step.